CriteoPublisherSdk/Sources/Standalone/CRBannerView.m

```diff
@@ -63,20 +63,42 @@ - (instancetype)initWithFrame:(CGRect)rect
                       webView:(WKWebView *)webView
                        adUnit:(CRBannerAdUnit *)adUnit
                     urlOpener:(id<CR_URLOpening>)opener {
+  return [self initWithFrame:rect
+                      criteo:criteo
+                     webView:webView
+                  addWebView:YES
+                      adUnit:adUnit
+                   urlOpener:opener];
+}
+
+- (instancetype)initWithFrame:(CGRect)rect
+                       criteo:(Criteo *)criteo
+                      webView:(WKWebView *)webView
+                   addWebView:(BOOL)addWebView
+                       adUnit:(CRBannerAdUnit *)adUnit
+                    urlOpener:(id<CR_URLOpening>)opener {
   if (self = [super initWithFrame:rect]) {
     _criteo = criteo;
     _webView = webView;
     _webView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
     _webView.scrollView.scrollEnabled = false;
     _webView.navigationDelegate = self;
     _webView.UIDelegate = self;
-    [self addSubview:webView];
+    if (addWebView) {
+      [self addSubview:webView];
+    }
     _adUnit = adUnit;
     _urlOpener = opener;
   }
   return self;
 }
 
+- (void)safelyLoadWebViewWithDisplayUrl:(NSString *)displayUrl {
```

Comment:
nit: `loadWebViewWithDisplayUrlSafely` (to conserve the convention of starting methods/functions with verbs).

Reply:
agreed but FAR as this naming is used across the codebase in a dozen place to wrap "safely" methods.